modules/extensions/k8stools.tf

```diff
@@ -23,14 +23,23 @@ resource "null_resource" "install_k8stools_on_operator" {
     content     = local.install_helm_template
     destination = "/home/opc/install_helm.sh"
   }
+
+  provisioner "file" {
+    content     = local.install_kubectx_template
+    destination = "/home/opc/install_kubectx.sh"
+  }
+
   provisioner "remote-exec" {
     inline = [
-      "chmod +x $HOME/install_kubectl.sh",
-      "$HOME/install_kubectl.sh",
+      # "chmod +x $HOME/install_kubectl.sh",
```

Comment:
It's considered bad practice leaving commented out code in the source. If you need to know what is was, that's what git is for. 😁

Reply:
Fixed and did the same for other scripts.

Reply:
I'm not seeing the fix in this PR?